Wrappers/Python/ccpi/web_viewer/trame_viewer3D.py

```diff
@@ -52,6 +52,8 @@ def __init__(self, list_of_files=None):
         super().__init__(list_of_files=list_of_files, viewer_class=CILViewer)
 
         # Define attributes that will be constructed in methods outside of __init__
+        self.omin = None
```

Comment:
what does omin mean vs cmin?

Reply:
i did explain it somewhere in the code but it wouldve been better if those comments lived where theyre initialised, so i will move them

omin, omax = min and max for the opacity mapping - so sometimes these are min and max in scalar, sometimes in gradient, depending on the setting
cmin, cmax = min and max for colour mapping, for colour mapping we always use scalar.

Looking back, I couldve just saved the min and max for scalar and gradient separately, rather than for color / opacity separately. Probably would've been better, but not sure it's worth going back and changing

Reply:
Ahhh I like it, leave it as is for now, I think.

Reply:
Leaving unresolved so I remember to add the comments